website/docs/cairo-1/06-interacting-with-starknet/06-deploy-account.md

```diff
@@ -1,27 +1,45 @@
-# Creating account
+# Creating an account
 
 ## Overview
 
-In order to create a new account, you need to deploy an account contract. Starknet expects a certain interface from an account contract. [OpenZeppelin](https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo) and [ArgentX](https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/account/ArgentAccount.cairo) provide account contract implementations that you can use.
+In order to create a new account, you need to deploy an account
+contract.
+
+There are multiple account contracts to choose from, and it is the end user responsibility to find an account that works
+for them. Some examples of already existing account contracts
+are [OpenZeppelin](https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo)
+and [ArgentX](https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/account/ArgentAccount.cairo).
```

Comment:
I'll rework this section entirely I think, I've kind of forgot to. 

But I'd still want this to be a "lower level" guide since onboarding through existing wallet app is simple enough IMO and explained in respective docs.

Reply:
> But I'd still want this to be a "lower level" guide since onboarding through existing wallet app is simple enough IMO and explained in respective docs.

Agreed, i would just like to have information about simpler optioned mentioned somewhere in the protostar docs
#1444 relevant issue

Reply:
I think we can resolve #1444 in a separate PR.